SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -794,11 +797,19 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 				}
 			}
 
+			// SpatialGDK - Creation of new entities should always be handled and therefore is checked prior to actor throttling.
+			// There is a EntityCreationRateLimit to prevent overloading spatial with creation requests if the developer desires.
+			// Creation of a new entity occurs when the channel is currently nullptr or if the channel does not have bCreatedEntity set to true.
+			if (FinalCreationCount < NumEntitiesToCreate && (Channel == nullptr || (Channel != nullptr && !Channel->bCreatedEntity)))
```

Comment:
Should we create an entity if the actor is marked as tear off from the very beginning? What does Unreal does natively?

Reply:
Ran a test with actors marked as torn off from begin play. It seems that the server never replicates these to the clients. To match this I've added an additional clause to prevent this from happening in Spatial too.